From a8be474648d620d6c8602055671e9d0194d1c509 Mon Sep 17 00:00:00 2001
From: Sultanxda <sultanxda@gmail.com>
Date: Sun, 2 Aug 2015 19:57:27 -0700
Subject: [PATCH 011/211] SnapdragonCamera: Initialize overlay before
 control-by-intent

* initializeControlByIntent() expects mVideoMenu to not be null
  and initializeOverlay() initializes mVideoMenu when it isn't null
* Change the order the functions are called to prevent possible NPE

Change-Id: I6d240892f3bd318031983243760d87ab46b25edb
---
 src/com/android/camera/VideoUI.java | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/com/android/camera/VideoUI.java b/src/com/android/camera/VideoUI.java
index 3a21a060a..75c9ab1db 100755
--- a/src/com/android/camera/VideoUI.java
+++ b/src/com/android/camera/VideoUI.java
@@ -276,8 +276,8 @@ public class VideoUI implements PieRenderer.PieListener,
         });
 
         initializeMiscControls();
-        initializeControlByIntent();
         initializeOverlay();
+        initializeControlByIntent();
         initializePauseButton();
 
         mCameraControls = (CameraControls) mRootView.findViewById(R.id.camera_controls);
-- 
2.17.1

